### PR TITLE
Remove duplicate hooks

### PR DESCRIPTION
**Pull request by axelboc**
Just a bit of almost-Spring cleaning.

### CODE REVIEW

File: packages/lib/src/vis/hooks.ts [hunk 1]
```diff
@@ -18,29 +17,25 @@ import {
...
 }
 
-export function useFrameRendering(): void {
-  const [, setNum] = useState(0);
-
-  useFrame(() => {
-    setNum(Math.random());
-  });
-}
-
```
Comment (axelboc) [inline packages/lib/src/vis/hooks.ts:39]: Moving things around a little.

File: packages/lib/src/vis/hooks.ts [hunk 2]
```diff
@@ -18,29 +17,25 @@ import {
...
 export function useDomains(
-  valuesArrays: (NdArray<number[]> | number[])[],
+  valuesArrays: AnyNumArray[],
```
Comment (axelboc) [inline packages/lib/src/vis/hooks.ts:36]: This hook wasn't the one that was used, so it didn't get the typed arrays changes.

File: apps/storybook/src/Utilities.stories.mdx [hunk 3]
```diff
@@ -86,9 +86,9 @@ const visDomain2 = getVisDomain([null, 20], [0, 100]); // [0, 20]
 
 #### getSafeDomain
 
-Determine a domain that is safe for the visualization. This is typically called with a user-defined `customDomain`, or with a `visDomain` as returned by `getVisDomain()`.
-If the domain is determined to be unsafe, a safe domain based on `fallbackDomain` is returned along with an error object. Note that `fallbackDomain` is assumed to be safe.
-The domain is considered unsafe if it's inverted (`min > max`), or if the scale is log and at least one of the bounds is negative.
+Make a domain safe in a given scale. This is typically called with a user-defined `customDomain`, or with a `visDomain` as returned by `getVisDomain()`.
+If `domain` is determined to be unsafe, a safe domain based on `fallbackDomain` is returned along with an error object. Note that `fallbackDomain` is assumed to be safe in the given scale.
+The domain is considered unsafe if it's inverted (`min > max`), if the scale is log and at least one of the bounds is negative, etc.
```
Comment (loichuder) [inline apps/storybook/src/Utilities.stories.mdx:91]: Not a fan of the addtion of `etc` here.

Im my opinion, we should make clear which domains are unsafe. If more cases need to be added, you can convert this in a bullet list

File: packages/app/src/vis-packs/core/heatmap/hooks.ts [hunk 4]
```diff
@@ -1,5 +0,0 @@
-import { getSafeDomain, getVisDomain } from '@h5web/lib';
-import { createMemo } from 'react-use';
-
-export const useVisDomain = createMemo(getVisDomain);
-export const useSafeDomain = createMemo(getSafeDomain);
```
Comment (loichuder) [inline packages/app/src/vis-packs/core/heatmap/hooks.ts:5]: Well spotted :monocle_face: !

**Review by loichuder — APPROVED** [review]

File: apps/storybook/src/Utilities.stories.mdx (hunk 3 above)
Comment (axelboc) [reply apps/storybook/src/Utilities.stories.mdx:91]: Might add some tests as well actually, and update the examples, I think they're out of date.

**Review by loichuder — APPROVED** [review]
Excellent :+1:

**Pull request merged**